gpt_index/readers/download.py

```diff
@@ -18,12 +19,17 @@
 )
 
 
-def download_loader(loader_class: str) -> BaseReader:
+def download_loader(
+    loader_class: str, refresh_cache: Optional[bool] = False
+) -> BaseReader:
     """Download a single loader from the Loader Hub.
 
     Args:
         loader_class: The name of the loader class you want to download,
             such as `SimpleWebPageReader`.
+        refresh_cache: If true, the local cache will be skipped and the
+            loader will be fetched directly from the remote repo.
+
```

Comment:
can we cache the library.json file locally so we also don't have to make a requests call to get library.json?

Reply:
I feel like that file is changing so often that it doesn't make too much sense right now? The cache would often be out of date and lead to issues. The cache of the modules themselves already leads to some unintuitive bugs when they need to be fixed (e.g. the `verbose` things), but I think long-term, caching modules is good since they won't change too much.

Reply:
That's a fair point - but i'm imagining a case where the parser (and even the LLM API) is local, and there's no need to make an extraneous request just to download a loader.

A solution to this may be to look up the cached library.json if the class name exists, but fetch library.json (and refresh the library.json cache) if the class name doesn't exist in the cached library.json